nucypher/characters/lawful.py

```diff
@@ -1093,113 +1077,66 @@ def __init__(self,
                            **character_kwargs)
 
         if is_me:
-            # Learner
-            self._start_learning_now = start_learning_now
 
-            # Self-Health Checks
+            # Operating Mode
+            self.known_node_class.set_federated_mode(federated_only)
+
+            # Health Checks
             self._availability_check = availability_check
             self._availability_tracker = AvailabilityTracker(ursula=self)
 
             # Datastore Pruning
-            self.__pruning_task = None
-            self._prune_datastore = prune_datastore
+            self.__pruning_task: Union[Deferred, None] = None
             self._datastore_pruning_task = LoopingCall(f=self.__prune_datastore)
 
-        #
-        # Ursula the Decentralized Worker (Self)
-        #
-
-        if is_me and not federated_only:  # TODO: #429
+            # Decentralized Worker
+            if not federated_only:
 
-            # Prepare a TransactingPower from worker node's transacting keys
-            _transacting_power = TransactingPower(account=worker_address,
-                                                  password=client_password,
-                                                  signer=self.signer,
-                                                  cache=True)
+                # Prepare a TransactingPower from worker node's transacting keys
+                transacting_power = TransactingPower(account=worker_address,
+                                                     password=client_password,
+                                                     signer=self.signer,
+                                                     cache=True)
+                self.transacting_power = transacting_power
+                self._crypto_power.consume_power_up(transacting_power)
+                self._checksum_address = transacting_power.account
```

Comment:
Here's another area where a checksum address consistency check can be enforced.

Reply:
✔️

Reply:
Looks like this one gives us some trouble in tests: <https://app.circleci.com/pipelines/github/nucypher/nucypher/7018/workflows/c520d6c6-6b62-407f-9527-947013614d1e/jobs/119591>